File: models/Manager.js
const mongoose = require('mongoose');

const ManagerSchema = new mongoose.Schema({
  managerId: String,
  managerName: String,
  password: String,
  designation: String,
  section: String,
  appointment:String,
  allProductReport: Boolean,
  demandReceived: Boolean,
  issueProduct: Boolean,
}, { timestamps: true });

module.exports = mongoose.model('Manager', ManagerSchema);

File: app/src/main/java/com/example/ncc_inventory/manager.kt
package com.example.ncc_inventory

data class Manager(
    val managerId : String,
    val email : String,
    val managerName : String,
    val password : String,
    val designation : String,
    val section : String,
    val appointment : String,
    val remark : String,
    val allProductReport : Boolean,
    val demandReceived : Boolean,
    val issueProduct : Boolean
)